### PR TITLE
Time series

### PR DESCRIPTION
**Pull request by FlanFlanagan**
This adds the name of the time series listener to the listening function. Backwards compatible as it doesn't touch any other part of the code, and only d3ploy really utilizes the listeners right now.

### CODE REVIEW

File: tests/bear_deploy.py [hunk 1]
```diff
@@ -60,7 +62,7 @@ def tick(self):
             self.context.schedule_build(self, proto)
 
 
-    def extract_supply(self, agent, time, value):
+    def extract_supply(self, agent, time, value, commod):
```
Comment (gwenchee) [inline tests/bear_deploy.py]: should this function return ` commodity_supply`

Comment (FlanFlanagan) [reply tests/bear_deploy.py]: I'm not sure what you mean here. This function doesn't return anything.

File: src/toolkit/timeseries.h [hunk 2]
```diff
@@ -29,11 +29,11 @@ enum TimeSeriesType : int{
...
-    std::function<void(cyclus::Agent*, int, bool)>,
-    std::function<void(cyclus::Agent*, int, int)>,
-    std::function<void(cyclus::Agent*, int, float)>,
-    std::function<void(cyclus::Agent*, int, double)>,
-    std::function<void(cyclus::Agent*, int, std::string)>
+    std::function<void(cyclus::Agent*, int, bool, std::string)>,
+    std::function<void(cyclus::Agent*, int, int, std::string)>,
+    std::function<void(cyclus::Agent*, int, float, std::string)>,
+    std::function<void(cyclus::Agent*, int, double, std::string)>,
+    std::function<void(cyclus::Agent*, int, std::string, std::string)>
```
[the remark this answers is not in the document]
Comment (FlanFlanagan) [reply src/toolkit/timeseries.h]: just following the previous std, but I'm not sure it's harmful though.

**Comment by FlanFlanagan** [comment]
@gwenchee @jbae11 

This needs a review.

File: tests/test_bear_deploy.py [hunk 3]
```diff
@@ -31,7 +31,7 @@
     'config': {
      'NOInst': {
       'growth_commod': 'bears',
-      'growth_rate': 0.05,
```
Comment (jbae11) [inline tests/test_bear_deploy.py]: I think these input parameters have to be updated to match the updated input parameters for NOInst

File: tests/test_bear_deploy.py [hunk 4]
```diff
@@ -19,8 +19,8 @@
    'config': {
     'DemandFac': {
      'commodity': 'bears',
-     'production_rate_max': 12,
-     'production_rate_min': 8,
+     'production_rate_max': 10,
+     'production_rate_min': 10,
```
Comment (katyhuff) [inline tests/test_bear_deploy.py]: What's this about? Why is the max=min ?

**Pull request merged**